docs/reference/module-api/module-overview.md

```diff
@@ -689,23 +714,19 @@ module.exports = {
 };
 ```
 
-### `routes(self)`
+If including a route option, use an object. See the [route options](#route-options) section for more.
 
-Add standard Express routes. The `routes` function takes takes the module as an argument and must return an object with properties for the relevant [HTTP request method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods), including `get`, `post`, `patch`, and `delete`. Each of those properties should be set to an object of functions. Route functions may be asynchronous (async).
```

Comment:
It is worth mentioning here that in most cases you will encounter fewer bugs if you use `apiRoutes` or `renderRoutes`, depending on your use case, and that classic Express routes are most useful if you need to use `res.redirect` or pipe a stream to `res`, two situations in which simply converting return values to responses and exceptions to status codes would be insufficient.

Reply:
We do have that note lower down: https://github.com/apostrophecms/a3-docs/pull/74/files#diff-65c9af67a92177dac44970f7f4e84f6d2a574d58e64e7b017a208bc3137d634eR750-R752